.github/workflows/ci.yml

```diff
@@ -152,13 +151,13 @@ jobs:
         os: [windows-latest, macos-latest, ubuntu-latest]
         include:
           - os: ubuntu-latest
-            gpu: 'yes'
+            gpu: "yes"
```

Comment:
Why change all of these? Having them differ from all of the other Linebender CI tasks would be annoying. We can change them everywhere, but would want to know why it is needed here.

Reply:
Just don't stage those changes in the PR then, please.

Reply:
fixed